### DISCUSSION TITLE
Some of MDI icons are not working

### DISCUSSION THREAD

**Discussion by vahem2lu** [requarks/wiki#6243] [Error / Bug Report] [answered]
Using version 2.5.297 and some of the MDI icons are not showing up
for example mdi-heat-wave
https://pictogrammers.com/library/mdi/icon/heat-wave/
<img width="503" alt="Screenshot 2023-03-09 at 20 52 34" src="https://user-images.githubusercontent.com/15649157/224126160-1579acc0-d8ea-4df7-8ed2-32d67a8887ee.png">

**Comment by NGPixel** [comment, marked as answer]
You need to use the icons for MDI v5. The `mdi-heat-wave` icon was added in v6.7.96 and therefore not available in Wiki.js.
